2025-02-22 16:08:57,057 INFO: Menu Dashboard startup
2025-02-22 16:09:01,863 INFO: Menu Dashboard startup
2025-02-22 16:09:01,863 INFO: Menu Dashboard startup
2025-02-22 16:11:20,821 INFO: Menu Dashboard startup
2025-02-22 16:11:25,179 INFO: Menu Dashboard startup
2025-02-22 16:11:25,179 INFO: Menu Dashboard startup
2025-02-22 16:18:42,284 INFO: Menu Dashboard startup
2025-02-22 16:18:46,650 INFO: Menu Dashboard startup
2025-02-22 16:18:46,650 INFO: Menu Dashboard startup
2025-02-22 16:20:10,990 INFO: Menu Dashboard startup
2025-02-22 16:20:11,704 INFO: Menu Dashboard startup
2025-02-22 16:20:11,704 INFO: Menu Dashboard startup
2025-02-22 16:21:41,524 INFO: Menu Dashboard startup
2025-02-22 16:21:42,298 INFO: Menu Dashboard startup
2025-02-22 16:21:42,298 INFO: Menu Dashboard startup
2025-02-22 16:22:09,421 ERROR: Exception on / [GET]
Traceback (most recent call last):
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Dev\HollyMenus\app\routes\main.py", line 93, in decorated_function
    return f(*args, **kwargs)
  File "C:\Dev\HollyMenus\app\routes\main.py", line 231, in index
    state = get_service_state()
            ^^^^^^^^^^^^^^^^^
NameError: name 'get_service_state' is not defined
2025-02-22 16:22:09,421 ERROR: Exception on / [GET]
Traceback (most recent call last):
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Dev\HollyMenus\app\routes\main.py", line 93, in decorated_function
    return f(*args, **kwargs)
  File "C:\Dev\HollyMenus\app\routes\main.py", line 231, in index
    state = get_service_state()
            ^^^^^^^^^^^^^^^^^
NameError: name 'get_service_state' is not defined
2025-02-22 16:33:06,762 INFO: Menu Dashboard startup
2025-02-22 16:33:09,341 INFO: Menu Dashboard startup
2025-02-22 16:33:09,341 INFO: Menu Dashboard startup
2025-02-22 18:35:55,106 INFO: Menu Dashboard startup
2025-02-22 18:35:57,016 INFO: Menu Dashboard startup
2025-02-22 18:35:57,016 INFO: Menu Dashboard startup
2025-02-22 18:39:59,358 INFO: Menu Dashboard startup
2025-02-22 18:40:01,235 INFO: Menu Dashboard startup
2025-02-22 18:40:01,235 INFO: Menu Dashboard startup
2025-02-22 18:42:46,678 INFO: Menu Dashboard startup
2025-02-22 18:42:50,144 INFO: Menu Dashboard startup
2025-02-22 18:42:50,144 INFO: Menu Dashboard startup
2025-02-22 18:43:33,285 INFO: Menu Dashboard startup
2025-02-22 18:43:34,954 INFO: Menu Dashboard startup
2025-02-22 18:43:34,954 INFO: Menu Dashboard startup
2025-02-22 18:44:04,305 INFO: Menu Dashboard startup
2025-02-22 18:44:05,852 INFO: Menu Dashboard startup
2025-02-22 18:44:05,852 INFO: Menu Dashboard startup
2025-02-22 18:45:36,316 INFO: Menu Dashboard startup
2025-02-22 18:45:37,984 INFO: Menu Dashboard startup
2025-02-22 18:45:37,984 INFO: Menu Dashboard startup
2025-02-22 18:51:48,064 INFO: Menu Dashboard startup
2025-02-22 18:51:51,129 INFO: Menu Dashboard startup
2025-02-22 18:51:51,129 INFO: Menu Dashboard startup
2025-02-22 19:03:14,647 INFO: Menu Dashboard startup
2025-02-22 19:03:16,558 INFO: Menu Dashboard startup
2025-02-22 19:03:16,558 INFO: Menu Dashboard startup
2025-02-22 19:09:07,909 INFO: Menu Dashboard startup
2025-02-22 19:09:12,179 INFO: Menu Dashboard startup
2025-02-22 19:09:12,179 INFO: Menu Dashboard startup
2025-02-22 19:16:58,171 INFO: Menu Dashboard startup
2025-02-22 19:16:59,821 INFO: Menu Dashboard startup
2025-02-22 19:16:59,821 INFO: Menu Dashboard startup
2025-02-22 19:40:14,650 INFO: Menu Dashboard startup
2025-02-22 19:40:17,076 INFO: Menu Dashboard startup
2025-02-22 19:40:17,076 INFO: Menu Dashboard startup
2025-02-22 19:50:05,069 INFO: Menu Dashboard startup
2025-02-22 19:50:06,928 INFO: Menu Dashboard startup
2025-02-22 19:50:06,928 INFO: Menu Dashboard startup
2025-02-22 19:53:07,402 INFO: Menu Dashboard startup
2025-02-22 19:53:09,368 INFO: Menu Dashboard startup
2025-02-22 19:53:09,368 INFO: Menu Dashboard startup
2025-02-22 20:22:43,323 INFO: Menu Dashboard startup
2025-02-22 20:22:49,225 INFO: Menu Dashboard startup
2025-02-22 20:22:49,225 INFO: Menu Dashboard startup
2025-02-22 20:31:15,123 INFO: Menu Dashboard startup
2025-02-22 20:31:32,485 INFO: Menu Dashboard startup
2025-02-22 20:31:32,485 INFO: Menu Dashboard startup
2025-02-22 20:31:35,734 ERROR: Exception on / [GET]
Traceback (most recent call last):
  File "C:\Dev\HollyMenus\app\routes\main.py", line 290, in index
    email_active = redis_client.get('service_state') == b'true'
                   ^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Dev\HollyMenus\app\routes\main.py", line 144, in decorated_function
    return f(*args, **kwargs)
  File "C:\Dev\HollyMenus\app\routes\main.py", line 314, in index
    return render_template('index.html', error=str(e))
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\jinja2\environment.py", line 1295, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Dev\HollyMenus\app\templates\index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Dev\HollyMenus\app\templates\base.html", line 144, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^
  File "C:\Dev\HollyMenus\app\templates\index.html", line 332, in block 'content'
    <div class="health-status {{ 'good' if all([db_status, redis_status, smtp_status]) else 'warning' if any([db_status, redis_status, smtp_status]) else 'error' }}"></div>
    ^^^^^^^^^^^^^
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\jinja2\utils.py", line 92, in from_obj
    if hasattr(obj, "jinja_pass_arg"):
       ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'all' is undefined
2025-02-22 20:31:35,734 ERROR: Exception on / [GET]
Traceback (most recent call last):
  File "C:\Dev\HollyMenus\app\routes\main.py", line 290, in index
    email_active = redis_client.get('service_state') == b'true'
                   ^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Dev\HollyMenus\app\routes\main.py", line 144, in decorated_function
    return f(*args, **kwargs)
  File "C:\Dev\HollyMenus\app\routes\main.py", line 314, in index
    return render_template('index.html', error=str(e))
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\jinja2\environment.py", line 1295, in render
    self.environment.handle_exception()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Dev\HollyMenus\app\templates\index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Dev\HollyMenus\app\templates\base.html", line 144, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^
  File "C:\Dev\HollyMenus\app\templates\index.html", line 332, in block 'content'
    <div class="health-status {{ 'good' if all([db_status, redis_status, smtp_status]) else 'warning' if any([db_status, redis_status, smtp_status]) else 'error' }}"></div>
    ^^^^^^^^^^^^^
  File "C:\Dev\HollyMenus\venv\Lib\site-packages\jinja2\utils.py", line 92, in from_obj
    if hasattr(obj, "jinja_pass_arg"):
       ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'all' is undefined
